File: game-server/src/main/java/org/forfun/mmorpg/game/cross/event/RpcConnectedEvent.java
package org.forfun.mmorpg.game.cross.event;

import jforgame.commons.eventbus.BaseEvent;

public class RpcConnectedEvent implements BaseEvent {

    private int serverId;

    private int serverType;

    public static RpcConnectedEvent valueOf(int serverId, int serverType) {
        RpcConnectedEvent event = new RpcConnectedEvent();
        event.serverId = serverId;
        event.serverType = serverType;

        return event;
    }

    public int getServerId() {
        return serverId;
    }

    public void setServerId(int serverId) {
        this.serverId = serverId;
    }

    public int getServerType() {
        return serverType;
    }

    public void setServerType(int serverType) {
        this.serverType = serverType;
    }

}

File: game-server/src/main/java/org/forfun/mmorpg/game/cross/event/RpcConnectedEvent.kt
package org.forfun.mmorpg.game.cross.event

import jforgame.commons.eventbus.BaseEvent

class RpcConnectedEvent : BaseEvent {

    var serverId: Int = 0
    var serverType: Int = 0

    companion object {
        fun valueOf(serverId: Int, serverType: Int): RpcConnectedEvent {
            val event = RpcConnectedEvent()
            event.serverId = serverId
            event.serverType = serverType
            return event
        }
    }
}
